docs/reference-guides/providers.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 0
+---
+
+# CAPI Providers
+
+Remember that most Cluster API Providers are upstream projects maintained by the Kubernetes open-source community.
+
+## List of officially supported providers
+
+:::note
+This list is constantly evolving to reflect the ongoing development of the project.
+:::
+
+This is a list of the officially supported CAPI Providers by Turtles. These providers are covered by our test suite and we actively ensure that they work properly with the CAPI extension for Rancher.
+
+| Platform        | Code Name                      | Provider Type            | Docs                     |
+|-----------------|--------------------------------|--------------------------|--------------------------|
+| **RKE2**            | CAPRKE2                    | Bootstrap/Control Plane  | https://github.com/rancher-sandbox/cluster-api-provider-rke2 |
+| **AWS**         | CAPA                           | Infrastructure           | https://cluster-api-aws.sigs.k8s.io |
```

Comment:
We can also mention that vSphere is currently in experimental mode

Reply:
Good point. I added a table for providers in experimental mode.